packages/bson-transpilers/test/yaml/native.yaml

```diff
@@ -1463,6 +1549,7 @@ tests:
           csharp: new DateTime(1995, 12, 17, 3, 24, 0)
           javascript: new Date('Sun, 17 Dec 1995 03:24:00 GMT')
           object: new Date('Sun, 17 Dec 1995 03:24:00 GMT')
+          php: new \DateTime('1995-12-17 03:24:00', new \DateTimeZone('UTC'))
```

Comment:
Related to my previous comment, I'll note that Python _is_ specifying a timezone here. IIRC, that's because the ISODate type is specifically intended for applying UTC (see: https://www.mongodb.com/docs/manual/reference/method/Date/).

Reply:
Also Ruby and Go forcibly use UTC timezone.

Reply:
OK to resolve.

Reply:
Fixed by using UTCDateTime.

Reply:
OK to resolve.